src/Packs.sol

```diff
@@ -493,6 +498,7 @@ contract Packs is
     /// @dev Only callable by admin role
     /// @dev Emits a Withdrawal event
     function withdrawTreasury(uint256 amount) external nonReentrant onlyRole(DEFAULT_ADMIN_ROLE) {
+        if (amount == 0) revert Errors.InvalidAmount();
```

Comment:
I agree with it.

Lately I've been thinking about operational vs user transactions. This is used so infrequently I don't think it matters.